Erreur inconnue (veuillez faire un rapport de bug) : <type 'exceptions.AttributeError'>\n
Sa valeur est : AttributeError("'plm.universe.bat.BatEntity' object has no attribute 'runI'",)
  File "<script>", line 13, in <module>
